distributor-api-v1/operations.md

```diff
@@ -1277,3 +1279,131 @@ In case of an error caused by insufficient availability \(which might have decre
 - `Charged` - Finite state. Payment has been successfully charged.
 - `Canceled` - Finite state. Payment has been canceled, and it has not been charged.
 - `Failed` - Finite state. Payment has not been charged.
+
+## Get Payment Cards <a id="get-payment-cards"></a>
+
+### Request`[ApiBaseUrl]/api/distributor/v1/paymentCards/getAll` <a id="request-apibaseurlapidistributorv1paymentcardsgetall"></a>
+
+```json
+{
+    "Client": "My Client 1.0.0",
+    "PaymentCardIds": [
+        "dc2f8608-9d71-47fd-9d41-ad1a009913c6"
+    ]
+}
+```
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `PaymentCardIds` | array of string | required | Collection of unique identifiers of the cards. |
+
+### Response <a id="response-8"></a>
+```json
+{
+    "PaymentCards": [
+        {
+            "Id": "dc2f8608-9d71-47fd-9d41-ad1a009913c6",
+            "AuthorizationState": "Authorizable"
+        }
+    ]
+}
+```
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `PaymentCards` | array of [PaymentCard](operations.md#payment-card)s | required | Payment cards that has been requested. |
```

Comment:
Payment cards that has been requested. -> `The payment cards.`? It's obvious that the developer requested them :)